client/python/lib/ovmsclient/tfs_compat/grpc/tensors.py

```diff
@@ -14,11 +14,52 @@
 # limitations under the License.
 #
 
+from tensorflow.core.framework.tensor_shape_pb2 import *
```

Comment:
Is it good idea to import everything? What do we use from tensor_shape_pb2 anyway? @mkuczyns11

Reply:
I investigated that and it seems, that we are using only TensorShapeProto.